code/modules/client/preferences.dm

```diff
@@ -1414,12 +1528,13 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 								organ_data[third_limb] = null
 								rlimb_data[third_limb] = null
 						if("Amputated")
-							organ_data[limb] = "amputated"
-							rlimb_data[limb] = null
-							if(second_limb)
-								organ_data[second_limb] = "amputated"
-								rlimb_data[second_limb] = null
-						if("Prothesis")
+							if(!(no_amputate))

```

Comment:
you remove the () from shit that needs it then add blatantly style-guide breaking ()s with NOT variable expressions